cobalt/BUILD.gn

```diff
@@ -13,6 +13,44 @@
 # limitations under the License.
 
 group("gn_all") {
+  testonly = true
+
   # TODO(b/371589344): Fix android build configs.
   deps = [ "//starboard($starboard_toolchain)" ]
+  if (!is_android) {
+    deps += [ ":cobalt" ]
+  }
+}
+
+if (!is_android) {
+  executable("cobalt") {
```

Comment:
GN-style I'd rather formulate this like:
```
executable("cobalt") {
  // TODO(...): Support Android TV (target_os == android),
  assert(!is_android);
```

to reduce the indentation.

Reply:
This is left under an if, to match content/shell/BUILD.gn, as for Android it's not an "executable" but a "group", and that will have to be brought in here.